include/graphics/primitives.h

```diff
@@ -5,8 +5,6 @@
 
 namespace arch {
 
-using Index = unsigned int;
```

Comment:
Maybe we should leave it here and use it in all other places? As it carries more meaning about our ontology.

Reply:
*ontology*
lol